ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -424,13 +433,15 @@ PrjFS_Result PrjFS_WritePlaceholderFile(
             &fileXattrData,
             PrjFSFileXAttrName))
     {
+        LogWarning("PrjFS_WritePlaceholderFile: failed to initialize placeholder for %s ", fullPath);
```

Comment:
I noticed several other spots where we're checking `InitializeEmptyPlaceholder` and not logging anything if it fails.  Could you update those spots as well?

Reply:
As discussed we logged inside InitializeEmptyPlaceholder to capture all places.